### ISSUE TITLE
Iron refractory iron deficiency anemia

### ISSUE THREAD

**Issue by GARDInfoCenter** [obophenotype/human-phenotype-ontology#3292]
While working on a case on iron refractory iron deficiency anemia, one of the information specialists provided feedback regarding the symptoms HPO lists for this disease. Some of the symptoms were thought to be a bit misleading and other symptoms listed were not found in any of the information resources.  

**Misleading symptoms???**
-Abnormality of the intestine (Is this the best symptom to list for diseases in which intestinal malabsorption is the actual finding?)
-Peripheral neuropathy (Is this listed because of the weakness that some of the patients experience? The weakness may not be from peripheral neuropathy, per se. Perhaps a different symptom would be better.)
-Ichthyosis (If a patient were to do an Internet search, one would pull up information about the genetic skin disorders. This could be disturbing to a patient.)

**Symptoms not found in our research**
-Intellectual disability
-Hypocupremia

Thank you,
Maria Della Rocca
Bilingual Information Specialist
maria.dellarocca@icf.com

**Comment by drseb** [comment]
Please explain exactly where you pull your data from. In our dataset I only see the associated symptoms:

```
"HP:0000007","Autosomal recessive inheritance"
"HP:0004840","Hypochromic microcytic anemia"
"HP:0001891","Iron deficiency anemia"
```

There must be some external data source that you are using.

**Comment by GARDInfoCenter** [comment]
I think those must be the ones being pulled in from Orphanet (IRIDA syndrome)
http://compbio.charite.de/hpoweb/showterm?disease=ORPHA:209981

**Comment by cmungall** [comment]
Thanks Maria,

Both orphanet and mondo treat  Orphanet:209981 and OMIM:206200 (http://compbio.charite.de/hpoweb/showterm?disease=OMIM:206200) as being equivalent, we'd like to get the HPO annotations for both to be consistent, correct and complete

**Comment by pnrobinson** [comment]
I have updated the annotations on our website. The annotation "HP:0001891","Iron deficiency anemia" is wrong and I removed it. The Orphanet annotation for Abnormality of the intestine is possibly coming from the reason that you said and we will report it to them, as it appears to be a mistake. We are feeding back to our colleagues at Orphanet and will report back to you shortly. Thanks for your feedback!
